src/docfx/validation/ContentValidator.cs

```diff
@@ -15,7 +15,7 @@ internal class ContentValidator
 
         public ContentValidator(Config config, FileResolver fileResolver, ErrorLog log)
         {
-            _validator = new Validator(GetMarkdownValidationRulesFilePath(fileResolver, config));
+            _validator = new Validator(GetValidationPhysicalFilePath(fileResolver, config.MarkdownValidationRules));
```

Comment:
```suggestion
            _validator = new Validator(fileResolver.ResolveFilePath(config.MarkdownValidationRules));
```

Reply:
will change that in another PR, need double confirm the `ResourceFilePath` behavior